testing/agent-for-testing/agent-for-testing.gradle

```diff
@@ -0,0 +1,80 @@
+plugins {
```

Comment:
This project is essentially a "custom distro", with no instrumentation included, and the testing exporter included.